lib/puppet/pal/catalog_compiler.rb

```diff
@@ -9,11 +9,9 @@ module Pal
   # @api public
   class CatalogCompiler < Compiler
 
-    # @api private
     def catalog
       internal_compiler.catalog
     end
-    private :catalog
```

Comment:
This is not great as it makes the entire Catalog in Puppet API - and that is definitively not wanted.

Reply:
This and what is on line 12 should be left unchanged since a method is added to get the catalog as a data hash.

Reply:
Oh derp, I dropped a previous commit that had that in it.